src/formatDate.js

```diff
@@ -50,7 +50,83 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [ , , , fromSeparator ] = fromFormat;
```

Comment:
```suggestion
  const fromSeparator = fromFormat[3];
```
looks much more readable

Reply:
Done ✅

Reply:
Sorry. Incidentally requested review😅